### PR TITLE
PUBDEV-6682 - Model metrics for IRF MOJO

### PR DESCRIPTION
**Pull request by Pscheidl**
https://0xdata.atlassian.net/browse/PUBDEV-6682

These metrics are only for the IRF MOJO.

### CODE REVIEW

File: h2o-bindings/bin/gen_R.py [hunk 1]
```diff
@@ -198,8 +198,8 @@ def gen_module(schema, algo, module):
         yield "    beta_constraints <- as.h2o(beta_constraints)"
         yield "  }"
     elif algo == "generic":
-        yield "  # Required args: model_key"
-        yield "  if (is.null(model_key)) stop(\"argument 'model_key' must be provided\")"
+        yield "  # Required args: either model_key or path"
+        yield "  if (is.null(model_key) && is.null(path)) stop(\"argument 'model_key' or 'path' must be provided\")"
```
Comment (honzasterba) [inline h2o-bindings/bin/gen_R.py]: should the same be done for py?

Comment (Pscheidl) [reply h2o-bindings/bin/gen_R.py]: I'm doing that for all the algos in the very next PR. Will definitely be there.

Comment (Pscheidl) [reply h2o-bindings/bin/gen_R.py]: Python is more permissive and only checks against list of allowed algos, thus no hotfix is needed, but the check will be unified. There is a backend check as well btw.

Comment (Pscheidl) [reply h2o-bindings/bin/gen_R.py]: I'll backport the change here.

**Comment by Pscheidl** [comment]
Do not resolve the issue just yet, there is one more PR coming.

**Pull request merged**